panel/dist/css/progress.css

```diff
@@ -0,0 +1,209 @@
+:host(progress) {
+  appearance: none;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  border: none;
+  height: 20px;
+  background-color: whiteSmoke;
+  border-radius: 3px;
+  box-shadow: 0 2px 3px rgba(0, 0, 0, 0.5) inset;
+  color: royalblue;
+  position: relative;
+  margin: 0 0 1.5em;
+}
+
+progress[value]::-webkit-progress-bar {
+  background-color: whiteSmoke;
+  border-radius: 3px;
+  box-shadow: 0 2px 3px rgba(0, 0, 0, 0.5) inset;
+}
+
+progress[value] ::-webkit-progress-value {
+  position: relative;
+  background-size: 35px 20px, 100% 100%, 100% 100%;
+  border-radius: 3px;
+}
+
+progress.active:not([value])::before {
+  background-position: 10%;
+  animation-name: stripes;
+  animation-duration: 3s;
+  animation-timing-function: linear;
+  animation-iteration-count: infinite;
+}
+
+progress[value]::-moz-progress-bar {
+  background-size: 35px 20px, 100% 100%, 100% 100%;
+  border-radius: 3px;
+}
+
+progress:not([value])::-moz-progress-bar {
+  border-radius: 3px;
+  background: linear-gradient(
+      -45deg,
+      transparent 33%,
+      rgba(0, 0, 0, 0.2) 33%,
+      rgba(0, 0, 0, 0.2) 66%,
+      transparent 66%
+    )
+    left/2.5em 1.5em;
+}
+
+progress.active:not([value])::-moz-progress-bar {
+  background-position: 10%;
+  animation-name: stripes;
+  animation-duration: 3s;
+  animation-timing-function: linear;
+  animation-iteration-count: infinite;
+}
+
+progress.active:not([value])::-webkit-progress-bar {
+  background-position: 10%;
+  animation-name: stripes;
+  animation-duration: 3s;
+  animation-timing-function: linear;
+  animation-iteration-count: infinite;
+}
+
+progress.primary[value]::-webkit-progress-value {
+  background-color: #007bff;
+}
```

Comment:
Given the extent of changes, I would really suggest start using CSS variables to keep these constants organized, e.g:
```css
:host {
  --primary-bg-color: #007bff;
}

progress.primary[value]::-webkit-progress-value {
  background-color: var(--primary-bg-color);
}
```
This way you get a component private CSS variable. Then, when a value is shared between components, this can be generalized using layered stylesheets, like bokehjs does, though our CSS variable usage is still quite limited.